app/actions/local/group.ts

```diff
@@ -74,12 +62,9 @@ export const searchGroupsByNameInChannel = async (serverUrl: string, name: strin
  * @param groups Group[] - The groups fetched from the API
  */
 export const storeGroups = async (serverUrl: string, groups: Group[]) => {
-    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
-    if (!operator) {
-        throw new Error(`${serverUrl} operator not found`);
-    }
-
     try {
+        const {operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
```

Comment:
0/5 This is already on a try block, so should not be much problem, but for example with the clients, we usually separate it in a different try block, to keep things separated. Not sure if we want to go that direction here.

Reply:
I'm 0/5 - left it in the same try just to keep down on the lines of code; the catch will do the same thing in both. We can start doing something like `catch(e: NetworkError){ ...} catch(e: ClientError){ ... }` as well do to do different things for different errors down the track.

Reply:
Multiple catch is not a JS thing, afaik

Reply:
Hopefully soon! https://github.com/Microsoft/TypeScript/issues/20024 🤞